LDM-148.tex

```diff
@@ -1138,7 +1138,7 @@ \subsubsection{Identity Management}\label{identity-management}
 
 \subsubsection{ITC Provisioning and Management}\label{itc-provisioning-management}
 
-This service provides for acquisition, change and configuration management, and provisioning of information technology components, whether purchased ``bare metal'', provided by the NCSA Commons, or provided by agreements with outside providers.
+This service provides for acquisition, change and configuration management, and provisioning of information technology components, whether purchased ``bare metal'', provided by a "commons" at the US Data Facility, or provided by agreements with outside providers.
```

Comment:
The term “commons” here feels awkward. Perhaps “shared with multiple projects at the US Data Facility”, or similar? Or even “...held in common with..”?